extensions/resource-deployment/src/test/services/apiService.test.ts

```diff
@@ -5,11 +5,15 @@
 
 import 'mocha';
 import assert = require('assert');
-import { apiService } from '../services/apiService';
+import { apiService } from '../../services/apiService';
 
-suite('API Service Tests', function (): void {
-	test('getAzurecoreApi returns azure api', () => {
+describe('API Service Tests', function (): void {
+	it('get azurecoreApi returns azure api', () => {
 		const api = apiService.azurecoreApi;
 		assert(api !== undefined);
 	});
+	it.skip('get azdataApi returns azdata api', () => {
```

Comment:
From @Charles-Gagnon 
Why are you adding a skipped test?

Reply:
The test works locally on mac but fails for some reason on Linux due to the extension not having been activated when the test runs. Do you have a suggestion on how to force extension activation? Should I remove it?

Reply:
Yeah let's just remove it I think - the test isn't really doing anything useful anyways

Reply:
Removed